app/rest/Discogs.js

```diff
@@ -3,22 +3,21 @@ const userToken = 'QDUeFOZNwIwOePlxpVziEHzamhbIHUdfENAJTnLR';
 const key = 'EZaGPpKGBbTkjwmpjmNY';
 const secret = 'uluhDSPtelRtLUvjrvQhRBnNwpZMtkZq';
 
-function addToken (query, first = false) {
-  let newQuery = query + '&token=' + userToken;
+function replaceQuery(query, first) {
   if (first) {
-    return newQuery.replace('&', '?');
-  } else {
-    return newQuery;
+    return query.replace('&', '?');
   }
+  return query;
+}
+
+function addToken (query, first = false) {
+  let newQuery = query + '&token=' + userToken;
+  return replaceQuery(newQuery, first);
 }
```

Comment:
this function can be shortened as

```javascript
return first ? query.replace('&', '?'); : query;
```

Reply:
Thanks for the suggestion! I´ve removed the replaceQuery function and replaced it with 
`return first ? newQuery.replace('&', '?') : newQuery;`
in the addToken and addKeys functions(from what I´ve seen addKeys isn´t being used atm, idk if it´ll be needed in the future or if it can be removed)

Reply:
I'm not sure about `addKeys`, I'll need to check git blame and see the rest of the code to be sure what happened to it and if it can be removed.